flow/environment.py

```diff
@@ -265,6 +269,13 @@ def get_prefix(cls, operation, parallel=False, mpi_prefix=None, cmd_prefix=None)
         # Can change to get them from directives for future
         return prefix
 
+    @classmethod
+    def get_default_directives(cls):
+        EXECUTABLE.default = sys.executable
```

Comment:
The fact we have to use this hack to make template tests pass is unfortunately. Could you make a comment explaining why we do this for future developers.